docs/src/references/child-devices.md

```diff
@@ -0,0 +1,120 @@
+   "device-type": "<optional type of that child-device>",
+   "<capability 1>": <capability specific JSON object>,
+   "<capability 2>": <capability specific JSON object>,
+   // [...]
+}
+```
+
+Thereby the fields are as below:
+   * `device-name` is an optional human readble device-name, visible in the cloud. If the field is not contained in the JSON message the `childid` of the topic structure is used as device-name.
+   * `device-type` is an optional device-type string assigned to the cloud's child-device twin. If the field is not contained in the JSON message the value `thin-edge.io-child` is used as device-type string.
+   * each field `<capability i>` represents a capability (e.g. `configurations` for _Configuration management_). The format of each `capability specific JSON object` is specific to the coresponding capability. For details see the documentation for coresponding capabilitie's API (e.g. [Configuration files for child devices](../references/c8y-configuration-management.md#configuration-files-for-child-devices)). 
+     Section [thin-egde APIs provided on network](#2-thin-egde-apis-provided-on-network-and-3-associate-consumedprovided-to-child-device-twins) below lists all APIs provided by thin-edge for child-device support.
+
+Example:
+```json
+{
+   "device-name": "My Child-Device",
+   "device-type": "type1",
+   "configurations": [ // capabilities for feature Configuration Management 
+     {
+       "type": "foo.conf"
+     },
+     {
+       "type": "bar.conf"
+     }
+   ]
+}
+```
+
+Whenever that MQTT message is sent to the thin-edge device:
+  1) thin-edge assures the child-device twin exists, and creates it if it does not exist.
+  2) each software component that provides an included capability (e.g. the C8Y Configuration Plugin for `configurations`) takes care to define all necessary capabilities to the coresponding cloud's child-device twin (i.E. _supported operations_, and list of provided types as e.g. _configuration types_).
+
+As result of the child-device creation thin-edge publishs an empty to MQTT topic `tedge/meta/success/<childid>` on successs, or a message to `tedge/meta/failed/<childid>`. The message on `tedge/meta/failed/<childid>` can optional contain a fail reaon as plain text.
+
+### (2) thin-egde APIs provided on network<br/> and (3) associate consumed/provided to child-device twins
+
+Each thin-edge API that supports child-devices covers both aspects. As of now, the following APIs have child-device support:
+   * Measurements, see [Sending a measurements to a child-devices](../tutorials/send-thin-edge-data.md#sending-measurements-to-child-devices)
+   * Events, see [Sending an event to a child-device](../tutorials/send-events.md#sending-an-event-for-a-childexternal-device-to-the-cloud)
+   * Configuration Management, see [Configuration files for child devices](../references/c8y-configuration-management.md#configuration-files-for-child-devices)
+
```

Comment:
Also, you can link the Alarms doc as well.

Reply:
added.